master/getting-started/docker/installation/manual.md

```diff
@@ -1,101 +1,68 @@
 ---
-title:  Preparing the environment for Calico as a Docker network plugin
+title: Installing Calico for Docker
 ---
 
-The worked example in the _Calico as a Docker network plugin tutorial_ is run
-on two Linux servers that have a number of installation requirements.
+Calico's core components are shipped in a Docker container.
```

Comment:
Similarly, the line below
"The node run command starts Calico's core services in the calico/node docker container. Check that they are running:"

Could just be

"Check that the calico/node container is running:"